### ISSUE TITLE
[1.42]Prices are jumping when switch between limit and advanced orders pages

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#2810]
1. Place several limit orders
2. Switch between limit/advanced orders pages
3. Pay attention to prices in the Your orders table

**AR**: prices are jumping/blinking (looks like reverted)

**ER**: prices are fixed.

**Comment by fairlighteth** [comment]
Investigated it for a bit. Seems related to `isInverted` here:

```
        <styledEl.RateValue>
          <RateInfo
            prependSymbol={false}
            isInvertedState={[isInverted, setIsInverted]}
            noLabel={true}
            doNotUseSmartQuote
            isInverted={isInverted}
            rateInfoParams={rateInfoParams}
            opacitySymbol={true}
          />
        </styledEl.RateValue>
```

If you set `isInverted` to `false` you don't see the same issue.

**Comment by elena-zh** [comment]
Should be fixed in #2901
